docs/releasehistory.rst

```diff
@@ -7,17 +7,18 @@ Releases follow the ``major.minor.micro`` scheme recommended by `PEP440 <https:/
 * ``minor`` increments add features but do not break API compatibility
 * ``micro`` increments represent bugfix releases or improvements in documentation
 
-1.0.0
+0.2.0
 -----
 
-This is the first major release of the toolkit intended for production use.
+This version of the toolkit introduces many new features on the way to a 1.0.0 release.
 
 New features
 """"""""""""
 
-* Major overhaul of the `SMIRNOFF specification <https://open-forcefield-toolkit.readthedocs.io/en/topology/smirnoff.html>`_ and its XML representation
+* Major overhaul of the `SMIRNOFF specification <https://open-forcefield-toolkit.readthedocs.io/en/master/smirnoff.html>`_ and its XML representation
```

Comment:
Suggest we explicitly it's now the SMIRNOFF 0.2 spec.

Reply:
Good idea -- changed in 8d10fcc